pkg/security/probe/probe_linux.go

```diff
@@ -1533,13 +1534,12 @@ func NewProbe(config *config.Config, opts Opts) (*Probe, error) {
 	p.scrubber = procutil.NewDefaultDataScrubber()
 	p.scrubber.AddCustomSensitiveWords(config.Probe.CustomSensitiveWords)
 
-	resolvers, err := resolvers.NewResolvers(config, p.Manager, p.StatsdClient, p.scrubber, p.Erpc)
-	if err != nil {
-		return nil, err
+	resolversOpts := resolvers.ResolversOpts{
+		PathResolutionEnabled: opts.PathResolutionEnabled,
 	}
-	p.resolvers = resolvers
+	p.resolvers, err = resolvers.NewResolvers(config, p.Manager, p.StatsdClient, p.scrubber, p.Erpc, resolversOpts)
```

Comment:
we should put back the check of err here I think

Reply:
yes, nice catch